recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.8.1" %}
+{% set version = "1.9.1" %}
```

Comment:
Have you verified what feature changes were done ... not sure if the bump from 1.8 to 1.9 might inflict API changes?

Reply:
Verified in the [changelog](https://github.com/holoviz/geoviews/blob/master/CHANGELOG.md) that no breaking changes have been introduced since v1.8.1.